SR2020/2020-02-19-inactive-team-prompt.md

```diff
@@ -0,0 +1,41 @@
+---
+to: Teams deemed 'inactive'
+subject: Lack of activity in Student Robotics
```

Comment:
Is there a wording which sounds more friendly?

Reply:
This was the nicest I could come up with, albeit with very little thinking.